java/libphonenumber/src/com/google/i18n/phonenumbers/MetadataManager.java

```diff
@@ -121,21 +139,14 @@ static PhoneMetadata getAlternateFormatsForCountry(int countryCallingCode) {
   }
 
   private static void loadShortNumberMetadataFromFile(String regionCode) {
-    InputStream source = PhoneNumberMatcher.class.getResourceAsStream(
-        SHORT_NUMBER_METADATA_FILE_PREFIX + "_" + regionCode);
-    ObjectInputStream in = null;
-    try {
-      in = new ObjectInputStream(source);
-      CodedInputByteBufferNano byteBuffer = convertStreamToByteBuffer(in, BUFFER_SIZE);
-      PhoneMetadataCollection shortNumberMetadata = new PhoneMetadataCollection();
-      shortNumberMetadata.mergeFrom(byteBuffer);
-      for (PhoneMetadata metadata : shortNumberMetadata.metadata) {
+    String fileName = SHORT_NUMBER_METADATA_FILE_PREFIX + "_" + regionCode;
+    InputStream source = PhoneNumberMatcher.class.getResourceAsStream(fileName);
```

Comment:
PhoneNumberMatcher? I am confused that the PhoneNumberMatcher class has the short number metadata at all?